projects/dotcms-js/src/lib/core/dotcms-events.service.ts

```diff
@@ -88,6 +88,11 @@ export class DotcmsEventsService {
 }
 
 export interface EventTypeWrapper {
-    data: any;
+    data: DotEventData;
     eventType: string;
 }
+
+export interface DotEventData {
```

Comment:
is this only be used here? or it would be better to move it to a single file?

Reply:
I was talking with @freddyucv the other day about namespaces instead of interfaces... ideas?

Reply:
I guess, it's something to discuss on SU

Reply:
other than multiples includes not sure what advantage namespaces will provide, again move the models to a separate file and not as part of a `service` or `component`